src/shared/components/layouts/styles/release-info.scss

```
@@ -0,0 +1,10 @@
+.release-info {
+  &__release_number {
+    transition: filter linear 0.25s;
+    filter: blur(0);
+  }
+
+  &__placeholder {
+    filter: blur(0.1rem);
```

```suggestion
    will-change: filter;
    filter: blur(0.1rem);
```
As filter is not necessarily hardware accelerated (even less so a blur which is one of the most complex available filter), I would signal to the browser that this will be animated with this `will-change` so that it has a chance to provide it with its own rendering layer. Not sure if the fact that blur effect bleeds outside of the element and needs information about the background would negate the improvements given by that, but still.

---

Cool, didn't know about that property